### PR TITLE
NVyrsta

### PR DESCRIPTION
**Pull request by NVyrsta**
[DEMO LINK](https://NVyrsta.github.io/react_static-list-of-posts/)

### CODE REVIEW

**Review by Malva37 — APPROVED** [review]
Nice job)

File: src/components/PostInfo/PostInfo.tsx [hunk 1]
```diff
@@ -1,5 +1,46 @@
...
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">
+          {title}
+        </h3>
+
+        <p>
+          {user && (
```
Comment (alexandra-protyanova) [inline src/components/PostInfo/PostInfo.tsx:28]: Let us don't render `p` tag if we don't have `user`

File: src/components/PostInfo/PostInfo.tsx [hunk 2]
```diff
@@ -1,5 +1,46 @@
...
+            </>
+          )}
+        </p>
+      </div>
+
+      <p className="PostInfo__body">
+        {body}
+      </p>
+
+      {comments.length > 0
```
Comment (alexandra-protyanova) [inline src/components/PostInfo/PostInfo.tsx:41]:
```suggestion
      {comments.length
```

**Review by alexandra-protyanova — APPROVED** [review]
GJ 💫

**Pull request closed without merging**